src/pages/SubmitFile.js

```diff
@@ -93,7 +93,7 @@ const SubmitFile = () => {
           {constants.SHOW_BEATCAPS && <BeatCapsInputs/>}
           <div className="file has-name">
             <label className="file-label" htmlFor="inputFile">
-              <input className="file-input" accept=".mp3,.mp4" id="inputFile" type="file" onChange={chooseFile} required/>
+              <input className="file-input" accept=".mp3,.mp4,.png,.jpeg" id="inputFile" type="file" onChange={chooseFile} required/>
```

Comment:
I think we should also include `.jpg` . I know it's silly that we have jpeg too but unfortunately file formats work this way :(